includes/class-dispatcher.php

```diff
@@ -125,7 +125,7 @@ public static function process_outbox( $id ) {
 	 *
 	 * @return array|void The next batch of followers to process, or void if done.
 	 */
-	public static function send_to_followers( $outbox_item_id, $batch_size = 50, $offset = 0 ) {
+	public static function send_to_followers( $outbox_item_id, $batch_size = ACTIVITYPUB_OUTBOX_PROCESSING_BATCH_SIZE, $offset = 0 ) {
```

Comment:
```suggestion
	public static function send_to_followers( $outbox_item_id, $batch_size = self::$batch_size, $offset = 0 ) {
```

Shouldn't it be `self::$batch_size` to be more consistent with the rest of the code?

Reply:
not possible :(